### PR TITLE
More compat with redshift

### PR DESCRIPTION
**Pull request by m-vdb**
Hey @shimizukawa,

I've forked your repo which helped me get started with Redshift + Django. I've made some more compat modifications that might interest you :)

### CODE REVIEW

File: django_redshift_backend/base.py [hunk 1]
```diff
@@ -35,9 +37,23 @@ def for_update_sql(self, nowait=False):
...
+    def _create_like_index_sql(self, model, field):
+        return None
+
+
 class DatabaseWrapper(BasePGDatabaseWrapper):
     vendor = 'redshift'
 
+    SchemaEditorClass = DatabaseSchemaEditor
+    data_types = deepcopy(BasePGDatabaseWrapper.data_types)
+    data_types["AutoField"] = "integer identity(0, 1)"
```
Comment (m-vdb) [inline django_redshift_backend/base.py]: `serial` doesn't exist in Redshift

File: django_redshift_backend/base.py [hunk 2]
```diff
@@ -35,9 +37,23 @@ def for_update_sql(self, nowait=False):
...
+    data_types["DateTimeField"] = "timestamp"
```
Comment (m-vdb) [inline django_redshift_backend/base.py]: `timestamp with timezone` doesn't exist in Redshift

File: django_redshift_backend/base.py [hunk 3]
```diff
@@ -35,9 +37,23 @@ def for_update_sql(self, nowait=False):
         raise NotImplementedError('SELECT FOR UPDATE is not implemented for this database backend')
 
 
+class DatabaseSchemaEditor(BasePGDatabaseSchemaEditor):
+
+    def _model_indexes_sql(self, model):
+        return []
+
+    def _create_like_index_sql(self, model, field):
+        return None
```
Comment (m-vdb) [inline django_redshift_backend/base.py]: no need for indexes

**Comment by shimizukawa** [comment]
Thanks for your PR! Just now I'm thinking redshift backend must support 'migrate' feature. I'm take a look the PR!

File: django_redshift_backend/base.py (hunk 1 above)
Comment (shimizukawa) [inline django_redshift_backend/base.py]: Typically auto increment id starts with 1 and my project uses `identity(1, 1)`.
Do you have any special reason to start from 0?

**Comment by shimizukawa** [comment]
LGTM with nits :)

**Comment by shimizukawa** [comment]
Now I'm going to merge this PR and I'll change `identity(0, 1)` to `identity(1, 1)`.

**Comment by shimizukawa** [comment]
Thanks!

File: django_redshift_backend/base.py (hunk 1 above)
Comment (m-vdb) [inline django_redshift_backend/base.py]: ah no, I should have put 1 :/

**Comment by m-vdb** [comment]
thanks @shimizukawa !

**Comment by m-vdb** [comment]
@shimizukawa do you think you can do a pypi release?

**Comment by m-vdb** [comment]
@shimizukawa can you do a pypi release for 0.3.0 ?

**Comment by shimizukawa** [comment]
OK, I will.

**Comment by m-vdb** [comment]
thanks :)

**Comment by shimizukawa** [comment]
https://pypi.python.org/pypi/django-redshift-backend/0.3

**Pull request merged**